source/admin_manual/list_of_events.rst

```diff
@@ -85,6 +85,12 @@ Root Categories
 +--------------------+---------------------------------------------------------+
 | smtp-submit        | SMTP submission client                                  |
 +--------------------+---------------------------------------------------------+
+| ssl                | SSL/TLS connections                                     |
++--------------------+---------------------------------------------------------+
+| ssl-client         | SSL/TLS connections originating from server             |
++--------------------+---------------------------------------------------------+
+| ssl-server         | SSL/TLS connections to the server                       |
```

Comment:
I'm not sure if these are wrong, or just confusing. How about:

 * ssl-client = Outgoing SSL/TLS connections
 * ssl-server = Incoming SSL/TLS connections